new/PRS13850WalattaS.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13850WalattaS" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Walatta Sanbat Tǝbarrǝh</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-08-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ወለተ፡ ሰንበት፡ ትበርህ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walatta Sanbat Tǝbarrǝh</persName>
+                    <persName xml:lang="gez" xml:id="n2">ወለተ፡ ሰንበት፡ ትበርሂ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Walatta Sanbat Tǝbarhi</persName>
+                    <persName xml:lang="gez" xml:id="n3">ወለተ፡ ሰንበት፡ ትበርሄ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n3">Walatta Sanbat Tǝbarhe</persName>
+                    <note>The last part of the name is attested throughout the manuscript with spelling variants.</note>
```

Comment:
As in another case previously, to me it seems reasonable to mention in which manuscript the various spellings are attested (although, as of yet, the person is of course only attested in one single manuscript... but we don't know what will come).

Reply:
I think it is ok, it will be linked to the manuscript :)